lib/improver/tests/metadata/test_check_datatypes.py

```diff
@@ -114,29 +113,12 @@ def test_float64_cube_coord_bounds_with_fix(self):
             np.array([(point - 10., point + 10.) for point in x_coord.points])
         )
         expected_cube = self.cube.copy()
-        expected_cube.coord("projection_x_coordinate").points = (
-            expected_cube.coord("projection_x_coordinate").points.astype(
-                np.float64))
         expected_coord = expected_cube.coord("projection_x_coordinate")
         enforce.check_cube_not_float64(self.cube, fix=True)
         self.assertEqual(self.cube, expected_cube)
         self.assertEqual(
             self.cube.coord("projection_x_coordinate"), expected_coord)
 
-    def test_float64_cube_time_coord_points_ok(self):
-        """Test a pass of a cube with 64 bit time coord points."""
-        self.cube.coord("time").points = (
-             self.cube.coord("time").points.astype(np.float64))
-        enforce.check_cube_not_float64(self.cube)
-
-    def test_float64_cube_forecast_ref_time_coord_points_ok(self):
-        """Test a pass of a cube with 64 bit fcast ref time coord points."""
-        frt_coord = iris.coords.AuxCoord(
-            [np.float64(min(self.cube.coord("time").points))],
-            standard_name="forecast_reference_time")
-        self.cube.add_aux_coord(frt_coord)
-        enforce.check_cube_not_float64(self.cube)
-
```

Comment:
Neither of these has any effect as they test things that are already on the cube in all the previous tests (ie the assignment statements on `self.cube` are either ineffectful or irrelevant).

Reply:
I agree that the tests for this function need some work. 

I guess the point of these particular tests were to show explicitly that time coordinates are ignored and don't raise an error. I know because of the way the test cubes are set up this behaviour is tested in the other tests, but not in such an obvious way. I personally don't think it hurts to show this behaviour explicitly rather than relying on knowledge of the setup cubes to know this has been tested. However I'm not going to object to you deleting them.